From 1f633bf2744a4b623a07f437f66f5d924778257f Mon Sep 17 00:00:00 2001
From: Zaikuo Wang <zaikuo.wang@intel.com>
Date: Wed, 17 Dec 2014 11:06:47 +0800
Subject: [PATCH 196/196] ov2722:[WORKAROUND]:change to another setting for
 480p video

CHT_RVP:video preview/recording hang in 480p resolution;
but the original setting on BYT_T work well
i'm not sure rootcause, but change to another setting can
work well on CHT_RVP
would revert this patch until find rootcause

Change-Id: I2a1c24a4abb5381f50638834e671084ad5099240
Tracked-On: https://jira01.devtools.intel.com/i#browse/IMINAN-12479
Signed-off-by: Zaikuo Wang <zaikuo.wang@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/33062
Reviewed-by: Tian Shu Qiu <tian.shu.qiu@intel.com>
Approver: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
---
 .../camera/drivers/media/i2c/ov2722.h              | 24 +++++++++++-----------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h b/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
index 0ea0033..5336dd1 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov2722.h
@@ -1121,18 +1121,18 @@ static struct ov2722_reg const ov2722_720p_30fps[] = {
 struct ov2722_resolution ov2722_res_preview[] = {
 	{
 		.desc = "ov2722_480P_30fps",
-		.width = 736,
-		.height = 496,
+		.width = 1632,
+		.height = 1092,
 		.fps = 30,
-		.pix_clk_freq = 73,
+		.pix_clk_freq = 85,
 		.used = 0,
-		.pixels_per_line = 2048,
-		.lines_per_frame = 1184,
+		.pixels_per_line = 2260,
+		.lines_per_frame = 1244,
 		.bin_factor_x = 1,
 		.bin_factor_y = 1,
 		.bin_mode = 0,
 		.skip_frames = 3,
-		.regs = ov2722_480P_30fps,
+		.regs = ov2722_1632_1092_30fps,
 	},
 
 	{
@@ -1241,18 +1241,18 @@ struct ov2722_resolution ov2722_res_video[] = {
 	},
 	{
 		.desc = "ov2722_480P_30fps",
-		.width = 736,
-		.height = 496,
+		.width = 1632,
+		.height = 1092,
 		.fps = 30,
-		.pix_clk_freq = 73,
+		.pix_clk_freq = 85,
 		.used = 0,
-		.pixels_per_line = 2048,
-		.lines_per_frame = 1184,
+		.pixels_per_line = 2260,
+		.lines_per_frame = 1244,
 		.bin_factor_x = 1,
 		.bin_factor_y = 1,
 		.bin_mode = 0,
 		.skip_frames = 3,
-		.regs = ov2722_480P_30fps,
+		.regs = ov2722_1632_1092_30fps,
 	},
 	{
 		.desc = "ov2722_720p_30fps",
-- 
1.9.1

